packages/core/src/data/sources/lazy/axisMeasureSource.js

```diff
@@ -0,0 +1,154 @@
+import SingleAxisLazySource from "./singleAxisLazySource.js";
+/**
+ * @typedef {import("../../../types/encoder.js").VegaScale} VegaScale
+ * @typedef {VegaScale & { props: import("../../../spec/scale.js").Scale }} ScaleWithProps
+ */
+
+/**
+ * Create measure data associated with the channel.
+ * Can be used to generate a visual clue ("measure") for the current zoom level.
+ */
+export default class AxisMeasureSource extends SingleAxisLazySource {
+    /** @type {number} */
+    startPos = 0;
```

Comment:
These could be class-private variables (i.e., start with \#).

On the other hand, you could have just a single variable that stores the object that you might publish when the domain changes. However, when propagating new data objects, it's best to always create a new object, not mutate an existing one.